src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
```

Comment:
I think we should move the stories next to the stories folder so they don't get built and shipped with the library.

Reply:
Actually, since we use `tsc` to build the whole `src/components` folder all stories will be included as well, I just pushed a commit to ignore the `**/**.stories.tsx` files and by that we can keep using the best practices you just mentioned :grin: